rbridge/rbridge/src/main/java/org/rbridge/RBridgeWorker.java

```diff
@@ -71,8 +71,6 @@ public JDefinedClass innerRun(Provider<ASTType> astTypeProvider) {
                 JDefinedClass innerBridge = rBridgeImpl._class(PUBLIC | STATIC | FINAL, innerName);
 
                 for (ASTField field : innerType.getFields()) {
-                    log.debug("Mapping " + target + "." + innerName + "." + field.getName() + " = " + id);
```

Comment:
I kind of like this debug logging... what do you think about added a flag to enable debug logging, effectively turning it off for the default case?